src/NorthwindWebsite/Business/Services/Implementations/ProductService.cs

```diff
@@ -26,6 +26,9 @@ public class ProductService : IProductService
         _supplierService = supplierService;
     }
 
+    public async Task<IEnumerable<Product>> GetAll() =>
+        await _productRepository.GetAll();
```

Comment:
Just curios: why don't you map your entity model to DTO model on Business layer?

Reply:
This is because out DTO has limitations on displaying products on a page. But api should return every product. So I'm just returning raw data from the database.